### DISCUSSION TITLE
Curious about decision to only allow hex 0-f for RGB values

### DISCUSSION THREAD

**Discussion by jsphcsnrs** [ixjf/MSIRGB#202] [General]
The hue wheel effect is a bit choppy and I'm wondering if this is because you can only step through colors in steps 1/16th the size of the color wheel instead of stepping through in 255ths. I noticed that this changed in v2, just curious why that was done.

**Comment by ixjf** [comment]
It wasn't. This is a hardware limitation. I just had it wrong in v1. Thought each byte-sized cell was one colour, but in fact it's 2 nibble-sized colours per cell.

**Reply by jsphcsnrs** [reply]
Gotcha, good to know. I've been playing with the script and my strips a bit and maybe there is a limitation in the strips that causes the very stark visual contrast between completely red, and very almost red.

**Comment by ixjf** [comment]
Yeah, I think it's a hardware problem too. There is one issue open, though, that claims that one setting that supposedly reverses colours also makes colours more accurate. I haven't tried this yet, but it's something that I'll look into when I can.
